packages/codebytes/src/types.ts

```diff
@@ -1,14 +1,15 @@
 import { LanguageOption } from './consts';
 
-interface CodebytesChangeHandler {
+export interface CodebytesChangeHandler {
   (text: string, language: LanguageOption): void;
 }
+
 export interface CodeByteEditorProps {
-  text: string;
-  language: LanguageOption;
-  hideCopyButton: boolean;
+  text?: string;
+  language?: LanguageOption;
+  hideCopyButton?: boolean;
   onCopy?: CodebytesChangeHandler;
-  isIFrame?: boolean;
+  isForums?: boolean;
```

Comment:
renaming this prop to just mention that the behavior is really specific to use in forums. figured it's better than 'isFrame'

Reply:
If this becomes open source then we will want it to be iFrame again, right?  Is there a plan for this to become open source?

Reply:
yeah we also may want to use codebytes on other non-discourse sites eventually! so +1 using a more generic name

Reply:
@j10sanders It is opensource as long at the client-modules repo is opensource. I'll change this to a more generic name, maybe isIFrame is better for now and if we decide to ever not use an iFrame in the plugin or on forums we can change!

Reply:
are you planning on changing the name in this pr?

Reply:
yes, i may have forgotten to do that here, let me update!